examples/aws-testing/cluster.lokocfg

```diff
@@ -29,11 +29,17 @@ cluster "aws" {
   dns_zone         = var.dns_zone
   dns_zone_id      = var.route53_zone_id
   ssh_pubkeys      = var.ssh_public_keys
+  os_channel       = "alpha"
```

Comment:
This changes the testing configuration to use `alpha` version instead of previously used `stable`? Is this intentional? What is the reasoning for that?

Even for testing setup, I don't think we should be promoting use of `alpha` channel.

Reply:
Yes, I used `alpha` for testing setups. Why not? I think Flatcar alpha/beta should be tested by Lokomotive users before it reaches stable.

Reply:
I think putting it here may result in many bug request reported to Lokomotive, which in fact will be caused by potential breakage in Flatcar. I'm not necessarily against it, I just think it should be a conscious decision. @iaguis what do you think?

Reply:
I can change it to `stable` for now until we have consensus. For me the main point was to make it obvious for users how to change the channels if they want to.